src/invidious/migration.cr

```diff
@@ -0,0 +1,38 @@
+abstract class Invidious::Migration
+  macro inherited
+    Invidious::Migrator.migrations << self
```

Comment:
migrations are automatically added to this list when the file is required which is why there's no need to do anything with migrations after they're defined.